eras/conway/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-conway`
 
+## Next version
+
+* Added `ToCBOR` and `FromCBOR` instances for `DefaultVote`.
+
```

Comment:
In a changelog

> New section is never added with the code changes. (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))

Instead, this line should be added to the top of the section for the latest version (i.e. `1.19.0.0`). Just above "Made the fields of predicate failures and environments lazy".

Reply:
Done, thanks!